docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+id: ga4gh_wes_sapporo_setup
+title: GA4GH WES Sapporo Setup
+---
+
+## Prerequisites
+
+You will need to have [Docker](https://docs.docker.com/get-docker/) installed for managing Sapporo
```

Comment:
**suggestion:** Can you specify the minimum version.  FYI, it's not `Docker version 24.0.5, build 24.0.5-0ubuntu1`

Reply:
Expected version number added in https://github.com/phac-nml/irida-next/pull/538/commits/3f2206b4ca63bcb6322baaf4c0d7725c73ed1fea